ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -163,7 +163,7 @@ export default {
       netRegexDe: NetRegexes.startsUsing({ id: '5EB1', source: 'Herzbube' }),
       netRegexFr: NetRegexes.startsUsing({ id: '5EB1', source: 'Jack' }),
       netRegexJa: NetRegexes.startsUsing({ id: '5EB1', source: 'ジャック' }),
-      delaySeconds: (_data, matches) => matches.duration - 6,
+      delaySeconds: (_data, matches) => matches.castTime - 6,
```

Comment:
Knockbacks I usually only give 5 seconds for, since the knockback ability usually is a different (uncasted) ability than the cast that causes it.

```suggestion
      delaySeconds: (_data, matches) => parseFloat(matches.castTime) - 5,
```

Reply:
This wasn't meant to be a logic change, just a bug fix?

Reply:
Ok, that's fair, then leave the 6, but I think this should have a `parseFloat`, imo.  Even if JavaScript allows "numbery string - number = number", I think it's a bit weird.